docs/source/en/api/models/autoencoder_dc.md

```diff
@@ -0,0 +1,56 @@
+<!-- Copyright 2024 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License. -->
+
+# AutoencoderDC
+
+*The 2D Autoencoder model used in [SANA](https://arxiv.org/papers/2410.10629) and introduced in [DCAE](https://huggingface.co/papers/2410.10733) by authors Junyu Chen, Han Cai, Junsong Chen, Enze Xie, Shang Yang, Haotian Tang, Muyang Li, Yao Lu, Song Han from MIT HAN Lab.*
+
+The following DCAE models are released and supported in Diffusers:
+- [`mit-han-lab/dc-ae-f32c32-sana-1.0`](https://huggingface.co/mit-han-lab/dc-ae-f32c32-sana-1.0)
+- [`mit-han-lab/dc-ae-f32c32-in-1.0`](https://huggingface.co/mit-han-lab/dc-ae-f32c32-in-1.0)
+- [`mit-han-lab/dc-ae-f32c32-mix-1.0`](https://huggingface.co/mit-han-lab/dc-ae-f32c32-mix-1.0)
+- [`mit-han-lab/dc-ae-f64c128-in-1.0`](https://huggingface.co/mit-han-lab/dc-ae-f64c128-in-1.0)
+- [`mit-han-lab/dc-ae-f64c128-mix-1.0`](https://huggingface.co/mit-han-lab/dc-ae-f64c128-mix-1.0)
+- [`mit-han-lab/dc-ae-f128c512-in-1.0`](https://huggingface.co/mit-han-lab/dc-ae-f128c512-in-1.0)
+- [`mit-han-lab/dc-ae-f128c512-mix-1.0`](https://huggingface.co/mit-han-lab/dc-ae-f128c512-mix-1.0)
+
+The models can be loaded with the following code snippet.
+
+```python
+from diffusers import AutoencoderDC
+
+ae = AutoencoderDC.from_pretrained("mit-han-lab/dc-ae-f32c32-sana-1.0-diffusers", torch_dtype=torch.float32).to("cuda")
```

Comment:
@lawrence-cj @chenjy2003 Will you be hosting the diffusers converted weights for the VAE as well?

Reply:
Oh yes. We will host the weight in the same place as the original ones.

Reply:
Cc: @chenjy2003 , please add our official diffusers version checkpoint into our huggingface collection.

Reply:
Sure. We can hold the converted weight. By the way, I noticed @a-r-r-o-w 's previous comments.

> cc @DN6 I think we could benefit from having .from_single_file loading for the autoencoders from this PR, because the original checkpoints have a good amount of downloads.

Could you please elaborate more? Do you mean that it is not necessary to hold the converted weights? Thanks in advance for your answer.

Reply:
Usually, whenever there is a conversion required to go from original implementation to diffusers version, we make sure to host the diffusers formats weights as well, so that it guarantees to work out of the box.

With single file loading (already added support for it in [this](https://github.com/huggingface/diffusers/pull/9708/commits/d6c748c7e665cbf0f4392dedd2b4b32253b3d462) commit, and you can find the example in docs as well as [here](https://github.com/huggingface/diffusers/pull/9708#issuecomment-2518441299)), one can load the original checkpoints directly as well. This, however, relies on the `config.json` file in the original repositories: https://huggingface.co/mit-han-lab/dc-ae-f128c512-mix-1.0/blob/main/config.json

```
{
  "model_name": "dc-ae-f128c512-mix-1.0"
}
```

Using this value, we choose the appropriate config for initalizing the VAE and rename the state-dict on the fly.

Reply:
I suppose what he mean is we can load the original weight using DCAE.from_single_file(xxx), but DCAE.from_pretrained(xxx) command still need a converted weight, right?

Reply:
Yes, `from_single_file` for original weights. `from_pretrained` for diffusers weights

Reply:
Got it. Thanks!